### ISSUE TITLE
Cover Shelving Unit Sharp Corners 

### ISSUE THREAD

**Issue by Stephanymtr** [bmhco/shared#29]
**To do:**
- [ ] Cover KALLAX shelving unit sharp corners 

Most of the furniture _in the playroom_ doesn't have sharp corners which makes the room super safe for little ones to run around. However, the KALLAX shelving unit we are currently using to display activities does have sharp corners. I think we should cover up those sharp edges by using corner protectors. 

<img width="1404" alt="Screenshot 2023-02-16 at 09 43 05" src="https://user-images.githubusercontent.com/91985721/219328378-93820505-b588-4074-863c-899e3d163800.png">

_I'm aware there is a plan for a custom shelving unit to be built for this space, so feedback on this particular task is welcome._

**Comment by nelsonic** [comment]
Good idea. 💡 👍 (thanks for opening this issue) 
I think @iteles has experimented with various corner protectors in the past and none of them last very long. 😖 
So ... I think maybe we need her feedback/insight on this 💭 
***and*** when we decide on the approach, we should use a stronger adhesive like [**`Epoxy Resin`**](https://www.leroymerlin.pt/v3/search/search.do?isautocompleted=true&autocomplete=top&keyword=epoxy%20cola) which we have in the workshop. 💪

**Comment by Stephanymtr** [comment]
_( I'm going over P-1 issues)_ @iteles do you have a preference on which corner protectors to use?

**Comment by nelsonic** [comment]
@Stephanymtr do you have any photos of these types of low-standing storage units at the Montessori Schools we visited? 📷 
_or_ have you been able to find anything online?
Curious how this is resolved by other established schools. 💭

**Comment by Stephanymtr** [comment]
No, I don't have photos. I remember taking photos/videos with A's phone. I don't know if it is still on the phone, but if I find something useful there, I will share it here.  I Googled photos of Montessori classrooms right now, and it seems that they chose furniture with rounded corners. I wasn't able to find Montessori classroom's photos of low-standing storage units (or any other furniture) with corner protectors. 

![image](https://user-images.githubusercontent.com/91985721/223961554-2a825c87-2d3a-42fd-8577-2c399f6d7acd.png)

**Comment by nelsonic** [comment]
OK the furniture has _rounded_ corners in some cases and exposed corners:

<img width="681" alt="image" src="https://user-images.githubusercontent.com/194400/223976750-0df50cda-0e08-46f7-986c-e19b0f160b0a.png">

We will ensure that any furniture we ***`make`*** will have rounded corners/edges e.g: https://github.com/dwyl/home/issues/190#issuecomment-1052675411

<img width="563" alt="image" src="https://user-images.githubusercontent.com/194400/223977552-6e61d525-d969-42ec-a3ab-0e0a30c1904f.png">

But for solving the _immediate_ problem of the `KALLAX` storage unit,
We should simply use a **_strong_ adhesive** and the corner protectors. 

@iteles do you have a preference for the corner protectors?

**Comment by iteles** [comment]
I'll check but I'm pretty sure we have a pack of corner protectors already. 

My preference is to use what we have!
